exercises/linked-list/.meta/src/reference/java/DoublyLinkedList.java

```diff
@@ -48,7 +48,7 @@ T shift() {
         private Element<T> prev;
         private Element<T> next;
 
-        Element(T value, Element<T> prev, Element<T> next) {
+        public Element(T value, Element<T> prev, Element<T> next) {
```

Comment:
I think I must have explained myself badly! I think that the nested class declaration on line 46 can remain `private` as it was originally, and the constructor method on line 51 can be made package-private.

Does this make sense? :slightly_smiling_face:

Reply:
Yes, is it because line 46 is declaring a generic type and 51 is creating a constructor for it?

Reply:
The class can remain private, as we prefer to [limit scope](https://github.com/exercism/java/blob/master/POLICIES.md#adhere-to-best-practices) , and this causes no problems.

The constructor method for the class can be reduced to package-private instead of public (as it was before your edits) as it isn't used outside of this package.